Add VECROS JETCORE-FC wiki page (#7810)
ArduPilot/ardupilot_wiki: common/source/docs/common-vecros-JetcoreFC.rst

.. _common-vecros-JetcoreFC:
[copywiki destination="copter,plane,rover,sub,blimp"]

> =========================

Reviewer: overline/underline too long...must match header width

   fan
-  Dimensions 115 x 80 x 45mm
-  Mount with 4 screws/spacers M3 109 x 74mm
-  WiFi module
-  WiFi antennas 2pcs

**Electrical parameters**

-  Power supply range: 12V – 35V (6S LiHV)
-  Integrated DC/DC converter for control circuits
-  Redundant power supply with power good monitoring for control unit
-  Current protected peripheral connectors
-  4 x FOC DroneCAN ESC 40A, featuring motor identification and motor
   diagnostics
-  Power sensor, SMBUS

**ArduPilot (Cube) Connections**

-  RC input
-  PWM output (7 + 7 lines)
-  2x CAN
-  4x UART
-  2x I2C
-  Buzzer
-  Power sensor input
-  ADC input

**Jetson connections**

-  PCI Express (M2, Key E connector)
-  4 USB 3.0 (ZIF connectors, reductions available)
-  Gigabit Ethernet (ZIF connector, reductions available)
-  6 CSI (22 pin)
-  4 GPIO
-  DroneCAN, UART, I2C, SPI
-  IMU BMI088 and barometer BMP388 on board
-  USB 2.0 for debugging
-  Micro SD card
-  Video output connector (micro HDMI)
-  Fan

**DroneCore.Power** (bottom power board)

-  4 x FOC DroneCAN ESCs – 40A peak / 20A continuous
-  power sensor, SMBUS,
-  LED Driver for 4x WS2812B strips

> **DroneCore.Pilot** is based on Cube Orange - the widely used autopilot by Hex.aero running on ArduPilot (or optionally PX4 flight stack) and Nvidia Jetson Xavier NX. Their rich interfaces allow connecting most sensors and other peripherals for drones available on the market.

Reviewer: ```suggestion
**DroneCore.Pilot** is based on Cube Orange - the widely used autopilot from CubePilot, running on ArduPilot (or optionally PX4 flight stack) and Nvidia Jetson Xavier NX. It's rich interfaces allow connecting most sensors and other peripherals for drones available on the market.
```

Author: changes are applied in the following commits

Author: please check and resolve
